src/main/java/com/rapid7/client/dcerpc/dto/SID.java

```diff
@@ -54,9 +54,6 @@ else if (identifierAuthority.length != 6) {
         }
         if (subAuthorities == null) {
```

Comment:
Technically we could initialize this to an empty array if null now

Reply:
@tosmun-r7 I prefer the caller to explicitly pass in an array then trying to be accommodating and assume they meant an empty array.